site/content/docs/developing/environment-variables.ja.md

```diff
@@ -27,7 +27,7 @@ database_name = os.getenv('DATABASE_NAME')
 * `COPILOT_ENVIRONMENT_NAME` - Service　が実行されている Environment 名(例: test、prod)
 * `COPILOT_SERVICE_NAME` - 現在の Service 名
 * `COPILOT_LB_DNS` - (存在する場合)ロードバランサー名。例: _kudos-Publi-MC2WNHAIOAVS-588300247.us-west-2.elb.amazonaws.com_ 注: カスタムドメイン名を利用している場合でも、この値はロードバランサーの DNS 名を保持します
-* `COPILOT_SERVICE_DISCOVERY_ENDPOINT` - サービス検出を介して、Environment の中で他の Service と通信するために Service 名の後に追加されるエンドポイント。値は `{env name}.{app name}.local` となります。サービスディスカバリについてのより詳しい情報は[サービス検出のガイド](../developing/service-discovery.ja.md) を参照してください
+* `COPILOT_SERVICE_DISCOVERY_ENDPOINT` - サービス検出を介して、Environment の中で他の Service と通信するために Service 名の後に追加されるエンドポイント。値は `{env name}.{app name}.local` となります。サービスディスカバリについてのより詳しい情報は[サービス検出のガイド](../developing/svc-to-svc-communication.ja.md#service-discovery) を参照してください
```

Comment:
We should be consistent with translation of the term "service discovery" used in the document, "サービス検出" or "サービスディスカバリ". I prefer "サービスディスカバリ" because it's more popular.

Reply:
Yes. I think consistency with translation is important,too. I will check the other documentation,  and try to make it "サービスディスカバリ"

Reply:
Thank you!